<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>282</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QListWidget" name="task_list"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_task"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button_done">
       <property name="text">
        <string>Terminer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button_add">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button_delete">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
